static/app/types/user.tsx

```diff
@@ -72,6 +72,7 @@ interface BaseApiToken {
   dateCreated: string;
   expiresAt: string;
   id: string;
+  name: string;
```

Comment:
```suggestion
  name?: string;
```

Names are optional right? This would break the contract for all past tokens since none of them would have any names

Reply:
Good catch!